.travis.yml

```diff
@@ -13,8 +13,5 @@ before_script:
 script:
   - vendor/bin/phpunit
 
-after_script:
-  - vendor/bin/coveralls
-
-notifications:
-  email: matthiasnoback@gmail.com
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I followed there guide and example
* https://github.com/codecov/example-php
* https://github.com/codecov/example-php/blob/master/.travis.yml